eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentRequest.scala

```diff
@@ -33,24 +33,29 @@ import scala.util.Try
  * Lightning Payment Request
  * see https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md
  *
- * @param prefix    currency prefix; lnbc for bitcoin, lntb for bitcoin testnet
- * @param amount    amount to pay (empty string means no amount is specified)
- * @param timestamp request timestamp (UNIX format)
- * @param nodeId    id of the node emitting the payment request
- * @param tags      payment tags; must include a single PaymentHash tag and a single PaymentSecret tag.
- * @param signature request signature that will be checked against node id
+ * @param prefix             currency prefix; lnbc for bitcoin, lntb for bitcoin testnet
+ * @param amount             amount to pay (empty string means no amount is specified)
+ * @param timestamp          request timestamp (UNIX format)
+ * @param nodeId             id of the node emitting the payment request
+ * @param tags               payment tags; must include a single PaymentHash tag and a single PaymentSecret tag.
+ * @param signature          request signature that will be checked against node id
+ * @param verifyFeatureGraph if false, doesn't validate that the feature bits are spec-compliant. This may be useful for
+ *                           backwards-compatibility with invoices that were generated before the spec made this mandatory.
  */
-case class PaymentRequest(prefix: String, amount: Option[MilliSatoshi], timestamp: Long, nodeId: PublicKey, tags: List[PaymentRequest.TaggedField], signature: ByteVector) {
+case class PaymentRequest(prefix: String, amount: Option[MilliSatoshi], timestamp: Long, nodeId: PublicKey, tags: List[PaymentRequest.TaggedField], signature: ByteVector)(verifyFeatureGraph: Boolean) {
```

Comment:
```suggestion
case class PaymentRequest(prefix: String, amount: Option[MilliSatoshi], timestamp: Long, nodeId: PublicKey, tags: List[PaymentRequest.TaggedField], signature: ByteVector) {
```

This is really a workaround for compatibility because we used to enable the MPP feature bit without the  `var_onion_optin`, which is now illegal. I would rather disable the feature bits dependency check for this particular feature, it has a low impact and saves us from circumvoluted changes in the codebase

Reply:
Fixed in `18a34ae`